fedimint-core/src/net/peers.rs

```diff
@@ -11,7 +11,6 @@ use crate::task::Cancellable;
 #[cfg(not(target_family = "wasm"))]
 pub mod fake;
 
-/// Owned [`PeerConnections`] trait object type
```

Comment:
Docs need fixing, need to point to `IPeerConnections`, but removing them seems unnecessary

Reply:
Again, docs seemed not that helpful here; kinda stating the obvious.